pkgs/top-level/all-packages.nix

```
@@ -39239,7 +39239,7 @@ with pkgs;
 
   xrq = callPackage ../applications/misc/xrq { };
 
-  pynitrokey = callPackage ../tools/security/pynitrokey { };
+  pynitrokey = python3Packages.callPackage ../tools/security/pynitrokey { };
```

```suggestion
  pynitrokey = callPackage ../tools/security/pynitrokey { };
```
We want to make it possible to override dependencies.

---

I didn't realise using callPackage over python3Packages.callPackage was preferential.